src/components/InputRange/index.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
```

Comment:
I would recomend to use next structure for react component.
Rename this file to input-range.component.tsx 
Add new file 'index.ts'

In new file 
```
import InputRange from './input-range.component';

export default InputRange;
```

It will be easier in the future to find the right tab in a large number